src/Query.test.tsx

```diff
@@ -42,7 +42,7 @@ describe("<Query />", () => {
   it("returns a component with the correct list of items", () => {
     usingMount(Component, (wrapper) => {
       const texts = wrapper.find("li").map(node => node.text());
```

Comment:
On line 6 - unused import spy, and stub from sinon can be removed
as well as lorem import on line 7

Reply:
removed